README.asciidoc

```diff
@@ -190,8 +190,8 @@ requests for consideration or create an issue with a code change proposal.
 === Installing dependencies
 
 On openSUSE one can install the package
-link:https://build.opensuse.org/project/show/devel:openQA[`os-autoinst-devel`] which
-pulls all dependencies.
+link:https://build.opensuse.org/project/show/devel:openQA[`os-autoinst-devel from devel:openQA`] which
```

Comment:
"os-autoinst-devel from " should not be part of the link.

Reply:
Why not?

Reply:
Because the link only points to the project page and not to the package page.

Reply:
ok. I am fine with either way